sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -221,6 +230,46 @@ void _CrtLeaveDbgReport(int reportType)
         _InterlockedDecrement(&_CrtInAssert);
 }
 
+EXTERN_C
+int __cdecl _CrtSetReportMode(int reportType, int reportMode)
+{
+    if (reportType >= _CRT_ERRCNT || reportType < 0)
```

Comment:
I believe the versions from MS do invoke the invalid error handler on invalid argument (both for reportType and reportMode), but I am fine with skipping that behavior for now.